src/open_inwoner/react/components/HomepagePluginSection/HomepagePluginSection.tsx

```diff
@@ -0,0 +1,58 @@
+import { ComponentChildren, AnyComponent as FC, RenderableProps } from 'preact';
+import { MaterialIcon } from '../MaterialIcon';
+import './HomepagePluginSection.scss';
```

Comment:
General comment about this directory - the item component in other dir. is named "HomePlugin" so without the word 'page for both the directory and the files. 
Not sure if that was a conscious choice, but you could remove the 'page' and all name it HomePluginSection, so it's a bit shorter and maybe less prone tot errors.